tests/foreman/cli/test_import.py

```diff
@@ -1338,3 +1404,112 @@ def test_reimport_enable_rh_repos_negative(self, test_data):
             )
         )
+
+    def test_import_content_hosts_default(self):
+        """@test: Import all content hosts from
+        the predefined dataset
+
+        @feature: Import Content-host
+
+        @assert: Profiles for all Content Hosts created
+
+        """
+        for test_data in gen_import_chost_data():
+            with self.subTest(test_data):
+                tmp_dir = self.default_dataset[0]
+                files = dict(self.default_dataset[1])
+                files = update_csv_values(files, test_data, tmp_dir)
+                # import the prerequisities and content hosts
+                imports = import_content_hosts(files, tmp_dir)
+                for entity in imports:
+                    self.assertEqual(imports[entity][0].return_code, 0)
+                # get the sat6 mapping of the imported organizations
+                imp_orgs = get_sat6_id(
+                    Import.csv_to_dataset([files['users']]),
+                    imports['organizations'][1]
+                )
+                # now to check whether all cont. hosts appeared insatellite
+                for imp_org in imp_orgs:
+                    self.assertNotEqual(
+                        ContentHost.list(
+                            {'organization-id': imp_org['sat6']}
+                        ).stdout,
+                        []
+                    )
+
+    def test_reimport_content_hosts_negative(self):
+        """@test: Repetitive Import of all content hosts from
+        the predefined dataset
+
+        @feature: Repetitive Import Content-host
+
+        @assert: Profiles for all Content Hosts created only once
+
+        """
+        for test_data in gen_import_chost_data():
+            with self.subTest(test_data):
+                tmp_dir = self.default_dataset[0]
+                files = dict(self.default_dataset[1])
+                files = update_csv_values(files, test_data, tmp_dir)
+                # import the prerequisities and content hosts
+                imports = import_content_hosts(files, tmp_dir)
+                for entity in imports:
+                    self.assertEqual(imports[entity][0].return_code, 0)
+                # get the sat6 mapping of the imported organizations
+                imp_orgs = get_sat6_id(
+                    Import.csv_to_dataset([files['users']]),
+                    imports['organizations'][1]
+                )
+                chosts_before = [
+                    ContentHost.list(
+                        {'organization-id': imp_org['sat6']}
+                    ).stdout
+                    for imp_org in imp_orgs
+                ]
+                reimport_chosts = Import.content_host_with_tr_data({
+                    u'csv-file': files['system-profiles'],
+                    u'export-directory': tmp_dir,
+                    u'verbose': True
+                })
+                self.assertEqual(reimport_chosts[0].return_code, 0)
```

Comment:
You're absolutely right,
this is something that was somehow 'lost in translation'. As the Exception raising in the Import class has been introduced to the code later.

I'll remove the redundant checks.